File: Models/Tables/PhraseMeaning.cs
public class PhraseMeaning
{
    public int Id { get; set; }
    public string? Meaning { get; set; }
    public string? Comment { get; set; }
    public int PhraseId { get; set; }

    public Phrase? Phrase { get; set; }
    public List<TagPhraseMeaning>? TagPhraseMeaning { get; set; } = new();
    public List<PhraseMeaningExample>? PhraseMeaningExample { get; set; } = new();
    public List<PhraseMeaningLexicon>? PhraseMeaningLexicon { get; set; } = new();
}

File: ClientApp/src/components/pages/PhrasesPage/PhrasePanel/PhraseMeaning.js
import React, {Component} from 'react';
import { Row, Col } from 'reactstrap';
import { PhraseExample } from './PhraseExample';
export class PhraseMeaning extends Component {
    render() {
        const phraseExamplesVector = this.props.phraseMeaning.phraseMeaningExamples.map(
            (phraseExample, index) => <PhraseExample 
                phraseExample={phraseExample} 
                key={index} 
            />
        );

        return (
            <li>
                <Row>
                    <Col className='phrase-meaning' xs="6">
                        <p><b>{this.props.phraseMeaning.meaning}</b></p>
                        <ul>{phraseExamplesVector}</ul>
                    </Col>
                    <Col className='phrase-comment'>
                        <p>{this.props.phraseMeaning.comment}</p>
                    </Col>
                </Row>
            </li>
        );
    }
}